lib/stack_master/commands/apply.rb

```diff
@@ -121,7 +121,7 @@ def stack_options
         {
           stack_name: stack_name,
           parameters: proposed_stack.aws_parameters,
-          capabilities: ['CAPABILITY_IAM'],
+          capabilities: ['CAPABILITY_IAM', 'CAPABILITY_NAMED_IAM'],
```

Comment:
Do we always want to blindly set these?  According to [the docs](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-iam-template.html#using-iam-capabilities), the intention was to make you think about what you're doing first.  Specifying these here circumvents that.